environment.yml

```
@@ -16,7 +16,7 @@ dependencies:
   - pint
   - pip
   - pip:
-      - git+https://github.com/libra-project/libra-toolbox@v0.3
+      - git+https://github.com/libra-project/libra-toolbox@v0.6.1
```

You should be able to install it directly from PyPi (ie not from git) now:

```suggestion
      - libra-toolbox>=0.6.1
```